README.md

```diff
@@ -141,17 +141,19 @@ The STL uses boost-math headers to provide P0226R1 Mathematical Special Function
 acquire this dependency.
 
 1. Install Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+2. Change directories to a location where you'd like a clone of this STL repository.
+3. Invoke `git clone https://github.com/microsoft/STL`
+4. Invoke `cd STL`
+5. Invoke `git submodule update --init -- vcpkg/`
+6. Invoke `cd vcpkg`. Alternatively, you can use a separate installation of vcpkg e.g. `https://github.com/microsoft/vcpkg`

```

Comment:
We've already linked vcpkg above, that's the `[vcpkg][]` citation, so we don't need to provide the URL here (which is confusing, since that URL isn't a "separate installation", it can just be used to obtain one).

Reply:
Ok I'll update accordingly.